contractcourt/contract_resolvers.go

```diff
@@ -20,6 +20,10 @@ const (
 	// sweepConfTarget is the default number of blocks that we'll use as a
 	// confirmation target when sweeping.
 	sweepConfTarget = 6
+
+	// secondLevelConfTarget is the confirmation target we'll use when
+	// adding fees to our second-level HTLC transactions.
+	secondLevelConfTarget = 6
```

Comment:
Good enough for now, but eventually this should start to be a function of the current height and the deadline to expiry (if the HTLC was contested).